NRediSearch/Schema.cs

```diff
@@ -129,5 +131,36 @@ public Schema AddSortableNumericField(string name)
             return this;
         }
 
+        public class TagField : Field
+        {
+            public string Separator { get; }
+            internal TagField(string name, string separator = ",") : base(name, FieldType.TAG, false)
+            {
+                Separator = separator;
+            }
+            internal override void SerializeRedisArgs(List<object> args)
+            {
+                base.SerializeRedisArgs(args);
+                if (!Separator.Equals(","))
```

Comment:
would prefer `if(Separator != ",")` here - avoids a possible NRE; note that C# has simpler operator usage than Java, so `==` does the obvious / correct thing